lib/web_ui/test/keyboard_converter_test.dart

```diff
@@ -51,7 +51,7 @@ void testMain() {
       return key.type == ui.KeyEventType.down;
     });
     bool preventedDefault = false;
-    final onPreventDefault = () { preventedDefault = true; };
+    final Null Function() onPreventDefault = () { preventedDefault = true; };
```

Comment:
Curious why you didn't use `VoidCallback`?

Reply:
Done.